packages/forms-web-app/src/views/rule-6/enter-code.njk

```diff
@@ -61,10 +61,41 @@
         }) }}
       </form>
 
-      <p class="govuk-body">
-        <a href="{{requestNewCodeLink}}" class="govuk-link">Send a new code</a>
+			<h2 class="govuk-heading-m">
+        If you’re having problems
+      </h2>
+
+      {% if confirmEmailLink %}
+        <p class="govuk-body">
+          <a href="{{confirmEmailLink}}" class="govuk-link">Change your email address</a> if you entered it incorrectly.
+        </p>
+      {% endif %}
+
+			<p class="govuk-body">
+        Check if the email has gone to your spam or junk folder. If it’s not there, we can <a href="{{requestNewCodeLink}}" class="govuk-link">send a new code</a>.
       </p>
 
+            <details class="govuk-details" data-module="govuk-details">
+        <summary class="govuk-details__summary">
+          <span class="govuk-details__summary-text">
+            If you’re still having problems
+          </span>
+        </summary>
+        <div class="govuk-details__text">
+          <p class="govuk-body">
+            You can contact our customer support team.
+          </p>
+
+          {# todo: use env variables for contact details #}
```

Comment:
is there a ticket in backlog for this? (should cover email address, telephone number, opening hours etc.)

Reply:
Not so far as I'm aware, but will check